### PR TITLE
Add Russian government services sites

### PR DESCRIPTION
**Pull request by WGOS**
This PR aims to add most commonly used government services sites in Russia.
It also includes a new category for those type of domains: `category-ru-gov`.

I hope that I've added everything in order so it could be accepted.

### CODE REVIEW

**Comment by IceCodeNew** [comment]
The whole structure looks fine to me. Though I'm quite curious what purpose does this PR serve?

**Comment by WGOS** [comment]
> The whole structure looks fine to me. Though I'm quite curious what purpose does this PR serve?

This is primarily for those Russian users who experience a lack of access to government sites using V2Ray/Xray proxy. Also Russian officials are announced that in case of "outer threat" they would shut down access to main sites from outside the country and as a consequence you won't be able to access them if your proxy server located let's say in the Netherlands.

**Comment by rootmelo92118** [comment]
> This is primarily for those Russian users who experience a lack of access to government sites using V2Ray/Xray proxy. Also Russian officials are announced that in case of "outer threat" they would shut down access to main sites from outside the country and as a consequence you won't be able to access them if your proxy server located let's say in the Netherlands.

Which ISP blocked Russian government website?

**Comment by WGOS** [comment]
> Which ISP blocked Russian government website?
The main point is to route those domains directly, not through proxy because they may be unreachable from outside the country

**Comment by rootmelo92118** [comment]
> > Which ISP blocked Russian government website?
> 
> The main point is to route those domains directly, not through proxy because they may be unreachable from outside the country

Ok. But, don't you want to add some kind of domain such as gov.ru?

**Comment by WGOS** [comment]
> > > Which ISP blocked Russian government website?
> > 
> > 
> > The main point is to route those domains directly, not through proxy because they may be unreachable from outside the country
> 
> Ok. But, don't you want to add some kind of domain such as gov.ru?

That's totally makes sense but for now I've just added most commonly used life-critical domains. Those sites listed in PR are used for receiving medical treatment, education management, extending your social transportation tickets etc.

Gov.ru domains, in other hand, mostly used to spread news of Russian ministries to the people. Most of all critical intercalations between the people and the government done via Gosuslugi (tr.: Government Services).

But I will for sure add `gov.ru` to this PR.
Should it be included to `category-ru-gov` or to its own file?

**Comment by rootmelo92118** [comment]
> > > > Which ISP blocked Russian government website?
> > > 
> > > 
> > > The main point is to route those domains directly, not through proxy because they may be unreachable from outside the country
> > 
> > 
> > Ok. But, don't you want to add some kind of domain such as gov.ru?
> 
> That's totally makes sense but for now I've just added most commonly used life-critical domains. Those sites listed in PR are used for receiving medical treatment, education management, extending your social transportation tickets etc.
> 
> Gov.ru domains, in other hand, mostly used to spread news of Russian ministries to the people. Most of all critical intercalations between the people and the government done via Gosuslugi (tr.: Government Services).
> 
> But I will for sure add `gov.ru` to this PR. Should it be included to `category-ru-gov` or to its own file?

You can add it in category-ru-gov directly.
But, my opinion is , you can open a new file to save every government's domain in Russia.
I saw your pull request, you store the Moscow government website in a separate file.
I think you can use one file to save it all.

**Comment by WGOS** [comment]
> > > > > Which ISP blocked Russian government website?
> > > > 
> > > > 
> > > > The main point is to route those domains directly, not through proxy because they may be unreachable from outside the country
> > > 
> > > 
> > > Ok. But, don't you want to add some kind of domain such as gov.ru?
> > 
> > 
> > That's totally makes sense but for now I've just added most commonly used life-critical domains. Those sites listed in PR are used for receiving medical treatment, education management, extending your social transportation tickets etc.
> > Gov.ru domains, in other hand, mostly used to spread news of Russian ministries to the people. Most of all critical intercalations between the people and the government done via Gosuslugi (tr.: Government Services).
> > But I will for sure add `gov.ru` to this PR. Should it be included to `category-ru-gov` or to its own file?
> 
> You can add it in category-ru-gov directly. But, my opinion is , you can open a new file to save every government's domain in Russia. I saw your pull request, you store the Moscow government website in a separate file. I think you can use one file to save it all.

Seprating them was done for the sake of modularity. But really I think that many users would add whole category to bypass proxy anyways so I'll merge everything into `category-ru-gov`

**Comment by KevinZonda** [comment]
Maybe more clear comment will be required?, Cuz it's pretty hard for us to know whats inside? Maybe I'll help

**Comment by WGOS** [comment]
> Maybe more clear comment will be required?, Cuz it's pretty hard for us to know whats inside? Maybe I'll help

So should I describe every domain listed in PR or you mean something else?

**Comment by KevinZonda** [comment]
> > Maybe more clear comment will be required?, Cuz it's pretty hard for us to know whats inside? Maybe I'll help
> 
> So should I describe every domain listed in PR or you mean something else?

Hi, would u mind to double check 3c932e4938f0434b79b2a81e593441020db448aa, are all descriptions proper?

**Comment by WGOS** [comment]
> > > Maybe more clear comment will be required?, Cuz it's pretty hard for us to know whats inside? Maybe I'll help
> > 
> > 
> > So should I describe every domain listed in PR or you mean something else?
> 
> Hi, would u mind to double check [3c932e4](https://github.com/v2fly/domain-list-community/commit/3c932e4938f0434b79b2a81e593441020db448aa), are all descriptions proper?

Mostly correct. I've brought `mosreg.ru` as a separate entity because of local territorial classifications

File: data/category-ru-gov [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Federal Public Services
+gosuslugi.ru
+gu-st.ru
+
+# Moscow Public Services
+mos.ru
+
+# Moscow Region (Province) Public Services
+mosreg.ru
```
Comment (KevinZonda) [inline data/category-ru-gov:13]:
```suggestion
# Public Services
mos.ru     #Moscow 
mosreg.ru  #Moscow Region (Province)
spb.ru     #St Petersburg
```

Would this be better? Humm

Comment (WGOS) [reply data/category-ru-gov:13]: Actually I could add all domains related to public services in Russia then

**Comment by WGOS** [comment]
I'll be working on adding bunch of regional sites / public services for now

**Comment by IceCodeNew** [comment]
> I'll be working on adding bunch of regional sites / public services for now

How about routing all sites with a .ru TLD?
I thought it might be enough for solving the problem.

**Comment by WGOS** [comment]
> > I'll be working on adding bunch of regional sites / public services for now
> 
> How about routing all sites with a .ru TLD? I thought it might be enough for solving the problem.

There are many .ru domains with DNS pointing to foreign IPs as well as many .ru domains are being blocked in Russia.
Russian IPs are also may be blocked

**Comment by IceCodeNew** [comment]
> > > I'll be working on adding bunch of regional sites / public services for now
> > 
> > How about routing all sites with a .ru TLD? I thought it might be enough for solving the problem.
> 
> There are many .ru domains with DNS pointing to foreign IPs as well as many .ru domains are being blocked in Russia.
> Russian IPs are also may be blocked

That's so bizarre. 🤯

**Comment by WGOS** [comment]
> > > > I'll be working on adding bunch of regional sites / public services for now
> > > 
> > > 
> > > How about routing all sites with a .ru TLD? I thought it might be enough for solving the problem.
> > 
> > 
> > There are many .ru domains with DNS pointing to foreign IPs as well as many .ru domains are being blocked in Russia.
> > Russian IPs are also may be blocked
> 
> That's so bizarre. 🤯

Yeah. This is feature of local law that even if you're hosting on a .ru domain and even on Russian IP, you will be blocked if you're violating censorship regulations. tjournal.ru is the most recent example though it is hosted on foreign IP.

**Comment by WGOS** [comment]
Guess it is done now. This is the most completed list I can find for now.

**Comment by IceCodeNew** [comment]
Maybe sort the list before merging it?

**Comment by WGOS** [comment]
> Maybe sort the list before merging it?

Maybe. But should it be sorted by the name of domain or subject?

**Comment by IceCodeNew** [comment]
> > Maybe sort the list before merging it?
> 
> Maybe. But should it be sorted by the name of domain or subject?

I would suggest you separate the list into several subjects (It seems you have done a great categorizing right now).
Then down to each subject, you could sort the domains under that subject by alphabetic order.

**Comment by WGOS** [comment]
> > > Maybe sort the list before merging it?
> > 
> > 
> > Maybe. But should it be sorted by the name of domain or subject?
> 
> I would suggest you separate the list into several subjects (It seems you have done a great categorizing right now). Then down to each subject, you could sort the domains under that subject by alphabetic order.

So shall it be sorted by Region, Territory, Autonomous Region and then each category by alphabetic order?

**Comment by IceCodeNew** [comment]
> > I would suggest you separate the list into several subjects (It seems you have done a great categorizing right now). Then down to each subject, you could sort the domains under that subject by alphabetic order.
> 
> So shall it be sorted by Region, Territory, Autonomous Region and then each category by alphabetic order?

Sounds good to me.
BTW, this PR is serving users like you, so it would be better to let you decide how to sort the categories in this list.

**Comment by WGOS** [comment]
Sorry for being abscent for quite some time. I've done some categorizing by statuses and sorting by an alphabetic order

**Review by KevinZonda — APPROVED** [review]
I believe this looks quite alright to me.

**Comment by KevinZonda** [comment]
> Sorry for being abscent for quite some time. I've done some categorizing by statuses and sorting by an alphabetic order

If u want make any further update, pls feel free to make another PR, thanx so much!

**Pull request merged**